packages/server-wallet/src/wallet/wallet.ts

```diff
@@ -52,24 +52,50 @@ export class Wallet {
             channelId: channelResult.channelId,
             currentStatus: newObjective.status,
             objectiveId: newObjective.objectiveId,
-            done: this.ensureObjective(newObjective, getMessages(createResult)).catch(error => ({
-              type: 'InternalError' as const,
-              error,
-            })),
+            done: this.ensureObjective(newObjective, getMessages(createResult)),
           };
         } catch (error) {
+          // TODO: This is slightly hacky but it's less painful then having to narrow the type down every time
+          // you get a result back from the createChannels method
+          // This should be looked at in https://github.com/statechannels/statechannels/issues/3461
           return {
-            channelParameters: p,
+            channelId: 'ERROR',
+            currentStatus: 'failed' as const,
+            objectiveId: 'ERROR',
             done: Promise.resolve({type: 'InternalError' as const, error}),
           };
         }
       })
     );
   }
 
+  /**
+   * Finds any approved objectives and attempts to make progress on them.
+   * This is useful for restarting progress after a restart or crash.
+   * @returns A collection of ObjectiveResults. There will be an ObjectiveResult for each approved objective found.
+   */
+  public async jumpStartObjectives(): Promise<ObjectiveResult[]> {
```

Comment:
Is it safe to call `jumpStartObjectives` during "normal operation"? Wouldn't it conflict with other ongoing operations?

Reply:
That's a good question! I don't think there should be a conflict as jumpstart is just essentially triggering messages to be resent. I don't think that would cause a conflict?

Perhaps worth writing a test?

Reply:
What kind of conflict could it create?

Reply:
I've added a test in 55a6c88a8842778e494179e13ef9b4fafeec8c3a that does a simple test of this behavior. I think this is a good question to keep in mind when designing an end-to-end test. 

If we do encounter an error with this I think a pretty simple solution is to only call `jumpstart` internally when creating the wallet.

Reply:
> If we do encounter an error with this I think a pretty simple solution is to only call jumpstart internally when creating the wallet.

Good idea! We would still need to return a handle on the ongoing objectives to the wallet user somehow.